pepper-apis/src/main/java/org/broadinstitute/ddp/util/PropertiesToMapTreeUtil.java

```diff
@@ -0,0 +1,55 @@
+package org.broadinstitute.ddp.util;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.TreeMap;
+
+import org.apache.commons.lang.StringEscapeUtils;
+
+/**
+ * Converts properties to hierarchy of maps: properties names are splitted into parts which separated by '.'
+ * and loaded to hierarchy of maps.
+ *
+ * <p>For example a template:
+ * <pre>
+ *     "Sections: $prequal.sect1.name, $prequal.sect2.name"
+ * </pre>
+ * and properties:
+ * <pre>
+ * Properties p = new Properties();
+ * p.put("prequal.sect1.name", "Sect_One");
+ * p.put("prequal.sect2.name", "Sect_Two");
+ * </pre>
+ * And the properties can be converted to a Map hierarchy and loaded to a Velocity context to evaluate a template.
+ */
+public class PropertiesToMapTreeUtil {
```

Comment:
The idea was to define a util class which could be reused in other classes (not only for Velocity engine).
Now it is used in VelocityUtil only, but in future it could be used in other code: may be to convert properties to maps could be helpful somewhere else..